packages/fdc3-standard/src/api/ContextMetadata.ts

```diff
@@ -18,3 +18,15 @@ export interface ContextMetadata {
    */
   readonly source: AppIdentifier;
 }
+
+export interface AppProvidableContextMetadata {
+  signature?: string;
```

Comment:
why are apps limited on the metadata they can send?  Shouldn't they be allowed to put anything in here?